src/cycax/cycad/cycad_part.py

```diff
@@ -175,5 +242,50 @@ def insert_hole(self, hole):
 
         self.features.append(hole)
         self.move_holes.append(hole)
-        
 
+    def side_location_calculator(self, side: str, x: float, y: float, sink=0):
+        """
+        Calculate given the side of and object and the relative x and y location on that side where the definite location is.
+            Args:
```

Comment:
Args/Raised/Returns are indented to much. Have a look at the docs it will display incorrectly.